astropy/io/fits/column.py

```diff
@@ -2010,6 +2010,9 @@ def __setitem__(self, key, value):
         np.ndarray.__setitem__(self, key, value)
         self.max = max(self.max, len(value))
 
+    def tolist(self):
+        return [ list(item) for item in super().tolist() ]
```

Comment:
Ok! Should E201 be added to code style then, eventually? Like, in a separate PR?

Reply:
If you hook up `flake8` to your IDE, the check is pretty strict, which would inform you of this:

https://github.com/astropy/astropy/blob/a9411363fa07346eaa821a842f8c8c645af9036d/setup.cfg#L152-L154

It is more laxed in CI for historical reasons:

https://github.com/astropy/astropy/blob/a9411363fa07346eaa821a842f8c8c645af9036d/tox.ini#L161

Reply:
thanks @pllim, yes, I have flake8 in my ide, but with even looser settings (for unrelated and possibly questionable reasons).

I did `tox -e codestyle` before committing. And it did not complain, as well as the CI. For the reason you mention, of course.
Which is why I meant to ask if maybe it would be useful for me to make an PR adding E201? 
Sorry, I do not know how style-only changes are handled here. Maybe it's too off-topic anyway.

Reply:
> make an PR adding E201

You are free to submit a PR, but you might also open a can of worms in case there are failures from code that was added like 10 years ago. 😬 

> how style-only changes are handled here

We currently handled it, uh, yolo-style? There is no style, but depends on who is looking at what, their mood, and how attentive they are to the diff. 😉

Reply:
Ah, but maybe adding E201 is much less controversial. I forgot we have that now.

https://github.com/astropy/astropy/blob/main/.pep8speaks.yml

Reply:
> > make an PR adding E201
> 
> You are free to submit a PR, but you might also open a can of worms in case there are failures from code that was added like 10 years ago. grimacing

I tried, and it's only 16 problems in total given by `tox -e codestyle`. I will make a PR, a bit later.

> 
> > how style-only changes are handled here
> 
> We currently handled it, uh, yolo-style? There is no style, but depends on who is looking at what, their mood, and how attentive they are to the diff. wink

Reply:
> primary style guidelines

Once in a while, we fantasize about making the tox check stricter but we don't want to deal with all the code that has been there many years. You brought up a good point about making this guideline clearer somewhere. Maybe such expectations could be made clearer over at https://docs.astropy.org/en/latest/development/codeguide.html#coding-style-conventions -- @embray ?

Reply:
> > primary style guidelines
> 
> Once in a while, we fantasize about making the tox check stricter but we don't want to deal with all the code that has been there many years. You brought up a good point about making this guideline clearer somewhere. Maybe such expectations could be made clearer over at https://docs.astropy.org/en/latest/development/codeguide.html#coding-style-conventions -- @embray ?

Thanks!

It says in there on the link to do `tox -e codestyle`, that's where I learned this is how to enforce guidelines. If it's ok, I will make the PR with adding E201 (and E202) and those style 16 fixes. I can deal with it later https://github.com/astropy/astropy/pull/12009 .